Add an example for restoring a searchable snapshot (#6002)
opensearch-project/documentation-website: _tuning-your-cluster/availability-and-recovery/snapshots/searchable_snapshot.md

:--- | :---
`storage_type` | `local` indicates that all snapshot metadata and index data will be downloaded to local storage. <br /><br > `remote_snapshot` indicates that snapshot metadata will be downloaded to the cluster, but the remote repository will remain the authoritative store of the index data. Data will be downloaded and cached as necessary to service queries. At least one node in the cluster must be configured with the `search` node role in order to restore a snapshot using the `remote_snapshot` type. <br /><br > Defaults to `local`.

> #### Example request

Reviewer: Add a space between lines 48 and 49.

> The following request restores `my-snapshot-name` as searchable snapshot.

Reviewer: ```suggestion
The following request restores the snapshot `my-snapshot-name` as a searchable snapshot:

```

````json
POST /_snapshot/my-repository-name/my-snapshot-name/_restore
{
  "storage_type": "remote_snapshot"
}

> ````

Reviewer 2: How about including a specific index to restore?

---

The following request restores `my-index` from `my-snapshot` as a searchable snapshot.
````json
POST /_snapshot/my-repository/my-snapshot/_restore
{
  "storage_type": "remote_snapshot",
  "indices": "my-index"
}

Author: I was about to add it with a specific index, then I thought it might be confusing for people who are restoring the whole snapshot. But both options works for me, I don't have strong opinion for one option over the other.

Author: Updated by adding the my-index

:--- | :---
`storage_type` | `local` indicates that all snapshot metadata and index data will be downloaded to local storage. <br /><br > `remote_snapshot` indicates that snapshot metadata will be downloaded to the cluster, but the remote repository will remain the authoritative store of the index data. Data will be downloaded and cached as necessary to service queries. At least one node in the cluster must be configured with the `search` node role in order to restore a snapshot using the `remote_snapshot` type. <br /><br > Defaults to `local`.

#### Example request

The following request restores the index `my-index` from the snapshot `my-snapshot` as a searchable snapshot:

````json
POST /_snapshot/my-repository/my-snapshot/_restore
{
  "storage_type": "remote_snapshot",
  "indices": "my-index"
}
````

> Just like any snapshot restore requests, you can include or exclude certain indexes or specify some other settings, see the [restore snapshots API]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore/#restore-snapshots) for more details.

Reviewer: ```suggestion
Similar to all snapshot restore requests, you can include or exclude certain indexes or specify additional snapshot settings. For more information, see the [restore snapshots API]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore/#restore-snapshots).
```
